documentation/scripts/buildSearchIndex.js

```diff
@@ -34,10 +34,15 @@ async function main() {
     const documents = [];
 
     // Components
-    for await (const path of globby.stream(
-        `${projectDir}/packages/*/README.md`
-    )) {
-        const componentName = /([^/]+)\/README.md$/.exec(path)[1];
+    for await (const path of globby.stream(`${projectDir}/packages/*/*.md`)) {
```

Comment:
Allow for all non-`CHANGELOG.md` markdown files to be read as "components" when building the docs.